### PR TITLE
Update guorui's manual testing

### LINKED ISSUE #244: Give manual testing for give, remove, done, undo, redo, edit and show

### PR DESCRIPTION
**Pull request by qinguorui2001**
fix #244

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1024,15 +1024,196 @@ testers are expected to do more *exploratory* testing.
...
+      shown in the status message. The assignment list panel will display **pending and completed** assignments sorted in 
+      due date with all pending assignments above completed ones.
+   
+   3. Test case: Repeat `give 1 d/lab2 by/11/11/2021` again.<br>
+      Expected: No assignment is added into the assignment list panel. Error details shown in the status message since no repeated assignments are allowed.
+   
+   4. Test case: `give 2 d/lab2 by/20/12/2021`<br>
+      Expected: "lab2" assignment is appended to second person's assignment list. Details of the added assignment will be
+      shown in the status message. The assignment list panel will display **second** person's **pending and completed** 
+      assignments sorted in due date with all pending assignments above completed ones.
```
Comment (yeo-yiheng) [inline docs/DeveloperGuide.md:1083]: "sorted **by** due date"

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1024,15 +1024,196 @@ testers are expected to do more *exploratory* testing.
...
+
+### Removing an assignment
+
+1. Removing an assigment while all assignments of a person are being shown.
+
+   1. Prerequisites: There are multiple persons in the contact list and the first person's assignments (at least one assignment) are shown already.
+
+   2. Test case: `remove 1`<br>
+      Expected: The first assignment is removed in assignment list panel. Details of the removed assignment will be
+      shown in the status message. The assignment list panel will display **pending and completed** assignments sorted in
```
Comment (yeo-yiheng) [inline docs/DeveloperGuide.md:1110]: "sorted by"

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1024,15 +1024,196 @@ testers are expected to do more *exploratory* testing.
...
+      Details of the edited person will be shown in the status message.
+   
+   3. Test case: `edit 2 n/Alex e/11465434@163.com m/CS2100` just after last test case.<br>
+      Expected: No person is edited. Error details shown in the status message since the repeated name is not allowed.
+   
+   4. Test case: `edit 2 n/Alex Yeoh e/15434@163.com m/CS2100` just after last test case.<br>
+      Expected: No person is edited. Error details shown in the status message since the repeated email is not allowed.
+   
+   5. Test case: `edit 2 t/friend t/lab7`<br>
+      Expected: The first person's tag is replaced with tags called "friend" and "lab7".
```
Comment (yeo-yiheng) [inline docs/DeveloperGuide.md:1177]: The command is `edit 2 ...` but you mentioned first person when it should be second.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1024,15 +1024,196 @@ testers are expected to do more *exploratory* testing.
...
+
+### Marking an assignment
+
+1. Marking an assigment while all assignments of a person are being shown.
+
+   1. Prerequisites: There are multiple persons in the contact list and the first person's assignments (at least one assignment) are shown already.
+
+   2. Test case: `done 1`<br>
+      Expected: The first assignment is marked as done in assignment list panel. Details of the marked assignment will be
+      shown in the status message. The assignment list panel will display **pending and completed** assignments sorted in
```
Comment (yeo-yiheng) [inline docs/DeveloperGuide.md:1197]: "sorted by"

**Review by yeo-yiheng — CHANGES_REQUESTED** [review]
Some changes to be made

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1024,15 +1024,196 @@ testers are expected to do more *exploratory* testing.
...
+   1. Prerequisites: The current assignment list panel is empty.
+   
+   2. Test case: `show`<br>
+     Expected: No assignments shown in assignment list panel. Error details shown in the status message since the format is invalid.
+   
+### Giving an assignment
+
+1. Giving an assigment while all assignments of a person are being shown.
+
+   1. Prerequisites: There are multiple persons in the contact list and the first person's assignments are shown already.
```
[the remark this answers is not in the document]
Comment (qinguorui2001) [reply docs/DeveloperGuide.md:1053]: good catch

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (qinguorui2001) [reply docs/DeveloperGuide.md:1160]: good catch

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (qinguorui2001) [reply docs/DeveloperGuide.md]: done

**Review by yeo-yiheng — APPROVED** [review]
LGTM!

**Pull request merged**